### ISSUE TITLE
Method possibly missing a [super awakeFromNib] call

### ISSUE THREAD

**Issue by puyanlin** [hoteltonight/HTCopyableLabel#4]
Is there any reason it should not call `[super awakeFromNib] ` in `awakeFromNib` ?
`- (void)awakeFromNib
{
    [self setup];
}
`

there is show this warning in latest Xcode version (8.3), can you help us fix this?
thank you very much!
I love this repo, it's help me a lot.

**Comment by sibljon** [comment]
I don't think so. If you submit a pull-request, we'll get it merged. Good call, and thanks @puyanlin !

**Comment by puyanlin** [comment]
ok, I just sent a pr for this issue, thank you.
<img width="1280" alt="screen shot 2017-04-05 at 12 24 26 pm" src="https://cloud.githubusercontent.com/assets/3139295/24689870/1c243720-19fc-11e7-8660-7ed8875844f4.png">
